docs/docs/code/bash/README.md

```diff
@@ -0,0 +1,199 @@
+# Writing Bash in Code Steps
+
+Prefer to write quick scripts in bash? We've got you covered.
+
+You can run any bash in a Pipedream step within your workflows.
+
+::: warning
+Bash steps are available in a limited alpha release.
+
+You can still run arbitrary bash scripts, including [sharing data between steps](/code/bash/#sharing-data-between-steps) as well as [accessing environment variables](/code/bash/#using-environment-variables).
+
+However, features available in [Node.js steps](/code/nodejs) like `$.respond`, `$.end`, and `$.auth` are not yet available in bash. If you have any questions please [contact support](https://pipedream.com/support).
+:::
+
+## Adding a Bash code step
+
+1. Click the + icon to add a new step
+2. Click "Custom Code"
+3. In the new step, select the bash runtime in language dropdown
+
+## Logging & debugging
```

Comment:
Maybe at the end of this section, note that "any standard output or standard error should also appear in the Logs section of the results"